.gitignore

```
@@ -9,4 +9,5 @@ gradle-app.setting
 .idea/
 local.properties
 maze_output/
-features/fixtures/rnapp/package-lock.json
+**/package-lock.json
+**/.cxx
```

I don't think the `**/` is necessary here

```suggestion
package-lock.json
.cxx
```

> A leading "\**" followed by a slash means match in all directories. For example, "**/foo" matches file or directory "foo" anywhere, the same as pattern "foo".
> https://git-scm.com/docs/gitignore